dask/dataframe/tests/test_dataframe.py

```diff
@@ -3546,6 +3552,7 @@ def test_autocorr():
 
 
 def test_apply_infer_columns():
+    warning = FutureWarning if PANDAS_GT_210 else None
```

Comment:
Nit: I would move this definition further down to just before it's used. This will, I think, make it easier for future readers to reason about.

Reply:
Updated